src/auto/about_dialog.rs

```diff
@@ -127,7 +127,6 @@ pub struct AboutDialogBuilder {
     parent: Option<Container>,
     receives_default: Option<bool>,
     sensitive: Option<bool>,
-    //style: /*Unknown type*/,
```

Comment:
Deprecated property

Reply:
Shouldn't we handle it like we do with functions? Like adding the deprecated attribute starting from a given version and remove it from the version it got removed?

Reply:
It was deprecated in 3.0: https://developer.gnome.org/gtk3/stable/GtkStyle.html#gtk-style-new
so it totally deprecated

Reply:
Perfect then!